citi-onboarding/maracatronics
# Feature/who we are section

## What i did
- Created the "Quem somos" section;
- Connected this section with Keystone;

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "feature/who-we-are-section" with ```git checkout feature/who-we-are-section```
3. Now, you need to open two terminals
4. In some terminal, go to client folder, with ```cd client```
5. In the other terminal, go to server folder with ```cd server```
6. In each terminal, run it with ```npm/yarn start``` in the client folder and ```nodemon index.js``` in the server.
7. Now, you need to open the http://localhost:3000/ (the client side)
8. In another page, open http://localhost:3001/admin 
9. Try to change some texts in "Quem somos" models and in "Propositos" models.
10. Reload the http://localhost:3000/ and check if the texts that you put in Keystone are there.
11. lastly, check if the section is responsive. You can use the dev tools.
